### ISSUE TITLE
Screw locations

### ISSUE THREAD

**Issue by rlaferla** [Eytecz/LGX_Lite_Stealthburner_CW2_style_mount#1]
Excellent mount but can you circle/show where the screws go?  Are there heat inserts too?

**Comment by rlaferla** [comment]
3 out of the 4 screw locations.  Are these correct?  What about the 4th?

![image](https://user-images.githubusercontent.com/1135694/155601472-27d53af1-1f77-4c64-8f0c-64fabeebbf95.png)

**Comment by Eytecz** [comment]
Hi, this seems a bit odd. I believe your LGX is not properly positioned? According to the CAD it should align perfectly. Keep in mind that you do not need to use those 2 holes for fixing the LGX. They are placed there in case you'd like to remove the LGX motor without completely disassembling the body.

![image](https://user-images.githubusercontent.com/94404962/155738501-4edda6c6-bbbf-4ec7-8a42-44499a48f384.png)

**Comment by rlaferla** [comment]
I'm sorry but I don't understand your answer.   I think my LGX Lite is positioned properly but the lever is just not in the upright position.  How is your printed part fixed to the LGX Lite?  Do I need to replace the two long black screws on the front of the LGX Lite with longer ones that also attach your mount?  ie.  the upper right and lower right diagonal screw holes.

**Comment by talmus-gibal** [comment]
If I'm understanding your question correctly, the LGX lite includes a pack of square M3 nuts. When you remove the front and back plates of the LGX lite you'll see a few slots around the perimeter of the main body which line up with the holes on the outside. Place four nuts on the bottom four slots then attach the two halves of the mount to the LGX lite.

**Comment by rlaferla** [comment]
Sorry for the delay but I am in the midst of a major overhaul to CAN bus umbilical and LGX Lite.  Now I'm back to this issue.  What I need (and another user posted a comment on another issue that they also need it) is a diagram or photo that shows where the screws and heat inserts go.  

An example would be this but just a simple photo where you marked the locations M3x12, heatinsert, etc... would suffice.  I know this seems obvious to you but we are asking because it's not obvious to us and we don't want to find out later that a screw was the wrong size or in the wrong location or needed a heat insert.

<img width="373" alt="image" src="https://user-images.githubusercontent.com/1135694/159315824-418e5605-441e-4a20-843c-4109efd5ca96.png">

**Comment by rlaferla** [comment]
Also, I reprinted the rear cover which fixed the alignment.  I may have used the wrong file initially.  I still need the screw types/length and heat insert locations

**Comment by rlaferla** [comment]
![image](https://user-images.githubusercontent.com/1135694/159334276-b5f3164e-ffe9-4a5a-8e18-9c2b8646c233.jpeg)![image](https://user-images.githubusercontent.com/1135694/159334295-9d04c721-b233-4eea-aaff-3145a2faad56.jpeg)

**Comment by rlaferla** [comment]
a. Holes 2 and 3 are not flush.  I don't know if the whole back needs to be flush/flat or if this is ok.
b. Hole 1, does this need a heat insert?
c. What are the pair of holes at 4 and 6 for?
d. What is 5?

![image](https://user-images.githubusercontent.com/1135694/159346038-37163568-6e0e-40ff-9ddc-095b965f3bc7.png)
